metpy/calc/thermo.py

```diff
@@ -267,9 +274,17 @@ def el(pressure, temperature, dewpt):
     ideal_profile = parcel_profile(pressure, temperature[0], dewpt[0]).to('degC')
     x, y = find_intersections(pressure[1:], ideal_profile[1:], temperature[1:])
 
-    # If there is only one intersection, it's the LFC and we return None.
+    # If there is only one intersection, there are two possibilities:
+    # the dataset does not contain the EL, or the LFC = LCL.
     if len(x) <= 1:
-        return np.nan * pressure.units, np.nan * temperature.units
+        if ((ideal_profile[-1] < temperature[-1]) & (len(x) == 1)):
```

Comment:
Just to clarify:
`and` -- logical and for `if`s and other such expressions
`&` -- bitwise and for arrays and integers.